src/components/views/room_settings/RoomPublishSetting.tsx

```diff
@@ -49,8 +58,10 @@ export default class RoomPublishSetting extends React.PureComponent<IProps, ISta
 
         client
             .setRoomDirectoryVisibility(this.props.roomId, newValue ? Visibility.Public : Visibility.Private)
-            .catch(() => {
+            .catch((err) => {
                 // Roll back the local echo on the change
+                console.log("here")
```

Comment:
I don't think you meant to leave this in ;)

Reply:
done